scholia/scrape/ojs.py

```diff
@@ -319,7 +319,20 @@ def _fields_to_content(fields):
     title = _fields_to_content(['citation_title', 'DC.Title',
                                 'DC.Title.Alternative'])
     if title is not None:
-        entry['title'] = title
+        original_title = title  # Store the original title
+        q = paper_to_q(entry)
+        if not q:  # If not identified before modification
+            # Replace dash or colon without altering surrounding spaces
+            title = title.replace(" –", ":").replace("– ", ":")
+            entry['title'] = title
```

Comment:
If there is no replacement, then it is not necessary to look up the second time.
```
if orginal_title != title:
```
could be included to avoid bothering the API a second time, - in most cases.

Reply:
So i should check if the paper is identified or not just one time?

Reply:
That depends, e.g., if the title is "An introduction to Scholia" and it does not exist in Wikidata, then doing a replacement would replace nothing (because there is no ":" or "-" in the title). So the second API lookup would return the same.

Reply:
Okay so if the title contains dash or colon the second API pass is okay ?